src/java/fr/paris/lutece/plugins/forms/web/admin/FormJspBean.java

```diff
@@ -456,6 +472,199 @@ public String getModifyForm( HttpServletRequest request ) throws AccessDeniedExc
+        model.put( MARK_EXPORT_CONFIG_LIST, ExportServiceManager.getInstance( ).createReferenceListExportConfig( formToBeModified, getLocale( ) ) );
+        
+        return getPage( PROPERTY_PAGE_TITLE_MODIFY_FORM, TEMPLATE_MANAGE_EXPORT, model );
+    }
+    
+    @Action( ACTION_CREATE_EXPORT_CONFIG )
+    public String doCreateExportConfig( HttpServletRequest request )
+    {
+        int nId = NumberUtils.toInt( request.getParameter( FormsConstants.PARAMETER_ID_FORM ), FormsConstants.DEFAULT_ID_VALUE );
+
+        if ( nId == FormsConstants.DEFAULT_ID_VALUE )
+        {
+            return redirectView( request, VIEW_MANAGE_FORMS );
+        }
+        
+        String field = request.getParameter( PARAMETER_EXPORT_CONFIG );
+        
+        Form formToBeModified = FormHome.findByPrimaryKey( nId );
+
+        if ( formToBeModified == null )
+        {
+            return redirectView( request, VIEW_MANAGE_FORMS );
+        }
+        
+        List<FormExportConfig> existingList = ExportServiceManager.getInstance( ).createReferenceListExportConfig( formToBeModified, getLocale( ) );
+
+        FormExportConfig config = new FormExportConfig( );
+        config.setIdForm( nId );
+        config.setField( field );
+        config.setOrder( existingList.size( ) + 1 );
+        
+        FormExportConfigHome.create( config );
+        
+        Map<String, String> mapParameters = new LinkedHashMap<>( );
+        mapParameters.put( FormsConstants.PARAMETER_ID_FORM, String.valueOf( nId ) );
+        
+        return redirect( request, VIEW_MANAGE_EXPORT, mapParameters );
+    }
+    
+    @View( VIEW_CONFIG_REMOVE_EXPORT_CONFIG )
+    public String getConfirmRemoveExportConfig( HttpServletRequest request )
+    {
+        int idConfig = NumberUtils.toInt( request.getParameter( PARAMETER_ID_CONFIG ), FormsConstants.DEFAULT_ID_VALUE );
+        int idForm = NumberUtils.toInt( request.getParameter( FormsConstants.PARAMETER_ID_FORM ), FormsConstants.DEFAULT_ID_VALUE );
+
+        if ( idConfig == FormsConstants.DEFAULT_ID_VALUE )
+        {
+            Map<String, String> mapParameters = new LinkedHashMap<>( );
+            mapParameters.put( FormsConstants.PARAMETER_ID_FORM, String.valueOf( idForm ) );
+            
+            return redirect( request, VIEW_MANAGE_EXPORT, mapParameters );
+        }
+
+        UrlItem url = new UrlItem( getActionUrl( ACTION_DELETE_EXPORT_CONFIG ) );
+        url.addParameter( PARAMETER_ID_CONFIG , idConfig );
+        url.addParameter(  FormsConstants.PARAMETER_ID_FORM , idForm );
+        
+        String strMessageUrl = AdminMessageService.getMessageUrl( request, MESSAGE_CONFIRM_REMOVE_EXPORT_CONFIG, url.getUrl( ), AdminMessage.TYPE_CONFIRMATION );
+
+        return redirect( request, strMessageUrl );
+
+    }
+    
+    @Action( ACTION_DELETE_EXPORT_CONFIG )
+    public String doDeleteExportConfig( HttpServletRequest request )
+    {
```

Comment:
Maybe you should add CSRF protections. It is a bit strange to confirm "remove" and then do "delete". Using a single word might be better.

Reply:
The second point was taken into account in the last commit.